### PR TITLE
Iteration 7 release docs

### PR DESCRIPTION
**Pull request by houli**
Also display the enclosing constructs in text form

### CODE REVIEW

File: doc/FEATURES_RELEASE_2.md [hunk 1]
```diff
@@ -53,11 +54,86 @@ must display useful error messages about such constructs to the user.
...
 Specifically, it should highlight name clashes for constructs named `clash1`
 and `clash2`.
 
+## Identify Sequential DDIs - Complete
+
+### Description
+Sequential DDIs arise because the PML contains a workflow in which the
+dispensing of one drug is always followed by the dispensing of another drug that
+has an interaction with the first. The system should identify these sequential
+DDIs and the enclosing `sequence` construct.
```
Comment (c-brenn) [inline doc/FEATURES_RELEASE_2.md]: Sequence isn't the only possible enclosing construct for a sequential DDI. Basically anything that isn't a branch or selection is possible.

File: doc/FEATURES_RELEASE_2.md [hunk 2]
```diff
@@ -53,11 +54,86 @@ must display useful error messages about such constructs to the user.
...
+
+### Testing
+Visit the [homepage]. Select `panacea/test/fixtures/ddis.pml` which contains a
+sequential DDI. Press `Submit`. The system should display the DDI information
+for the sequential DDI found in the analysis tab including its enclosing
+construct and the drugs involved.
```
Comment (c-brenn) [inline doc/FEATURES_RELEASE_2.md:75]: Do you it's worth including the specific DDI info that will be visible here? He seems to like a lot of specificity.

File: doc/FEATURES_RELEASE_2.md [hunk 3]
```diff
@@ -53,11 +54,86 @@ must display useful error messages about such constructs to the user.
...
+### Description
+Parallel DDIs arise because the PML contains interacting drugs in two different
+"arms" of a PML branch construct. The system should identify these parallel DDIs
+and the enclosing `branch` construct.
+
+### Testing
+Visit the [homepage]. Select `panacea/test/fixtures/parallel.pml` which contains
+a parallel DDI. Press `Submit`. The system should display the DDI information
+for the parallel DDI found in the analysis tab including its enclosing construct
+and the drugs involved.
```
Comment (c-brenn) [inline doc/FEATURES_RELEASE_2.md:91]: I suppose the same question here and in the other `Identify X DDI` sections.

File: doc/FEATURES_RELEASE_2.md [hunk 4]
```diff
@@ -53,11 +54,86 @@ must display useful error messages about such constructs to the user.
...
+Visit the [homepage]. Select `panacea/test/fixtures/repeated_alternative.pml`
+which contains a repeated alternative DDI. Press `Submit`. The system should
+display the information for the repeated alternative DDI found in the analysis
+tab including its enclosing construct and the drugs involved.
+
+## Specify a Delay - Complete
+
+### Description
+The system should provide a way to say, at any point in a sequential workflow,
+that the enactment of the pathway will pause/delay for a given time.
```
Comment (c-brenn) [inline doc/FEATURES_RELEASE_2.md:138]: Do you think we should include an example of what a delay looks like in our system? Similar to how what we did with `Specifying a drug in PML` for release 1.

**Review by c-brenn — CHANGES_REQUESTED** [review]
solid spelling 👁

File: doc/FEATURES_RELEASE_2.md [hunk 5]
```diff
@@ -87,6 +90,9 @@ a parallel DDI. Press `Submit`. The system should display the DDI information
 for the parallel DDI found in the analysis tab including its enclosing construct
 and the drugs involved.
 
+Specifically, it should show that the torasemide/trandolapril interaaction is a
```
Comment (c-brenn) [inline doc/FEATURES_RELEASE_2.md]: interaaction -> interaction

File: doc/FEATURES_RELEASE_2.md [hunk 6]
```diff
@@ -118,12 +127,39 @@ which contains a repeated alternative DDI. Press `Submit`. The system should
 display the information for the repeated alternative DDI found in the analysis
 tab including its enclosing construct and the drugs involved.
 
+Specifically, it should show that the torasemide/trandolapril interaction is an
```
Comment (c-brenn) [inline doc/FEATURES_RELEASE_2.md]: an -> a

**Comment by c-brenn** [comment]
Looks good. He didn't buy some of the `Identify X DDI` features that we've done - do you think we should highlight them as `bonus features` or something to that effect?

**Comment by c-brenn** [comment]
for potential extra marks

**Review by c-brenn — APPROVED** [review]

**Comment by 22a** [comment]
Solid work, chaps! This looks great!

**Pull request merged**